src/components/Molecules/HeaderEsuWithIcon/HeaderEsuWithIcon.style.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 import Icon from '../../Atoms/SocialIcons/Icon/Icon';
 import spacing from '../../../theme/shared/spacing';
-import EmailSignUp from '../EmailSignUp/EmailSignUp';
+import EmailSignUp from '../../Organisms/EmailSignUp/EmailSignUp';
```

Comment:
Is this not just a Component, to keep it inline with existing stuff? Or should we revisit naming?

Does it matter?

Does anything?

_Ad Astra_ messed me up, man.

Reply:
Ah, you already renamed it all already.

Tommy Lee Jones would be proud.